src/client/components/SocialLinks.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import SocialLink from './SocialLink';
+
+export const socialProfiles = [
```

Comment:
might be good a idea to keep `socialProfiles`, `socialTransformers` and `transform` in a helper / constants file instead of a components file